api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -9056,4 +9056,53 @@
 		<!-- visit_attribute_type has not null constraint on uuid -->
 		<!-- visit_type has not null constraint on uuid -->
 	</changeSet>
+
+    <changeSet id="TRUNK-3422-adding-Get-Visits-Privilege" author="PralayRamteke" >
+        <preConditions  onFail="MARK_RAN">
+            <not>
+                <sqlCheck expectedResult="0">
+                SELECT count(*) FROM role_privilege  WHERE privilege='View Encounters';
+                </sqlCheck>
+            </not>
+        </preConditions>
+        <comment>
+            Add "Get Visits" privilege to the roles having "View Encounter"
+        </comment>
+            <sql>
+                INSERT INTO role_privilege (role, privilege) 
+                SELECT role, 'Get Visits' from role_privilege rp
+                WHERE rp.privilege = 'View Encounters'
+                AND NOT EXISTS (SELECT role, privilege FROM role_privilege rp2 WHERE rp2.privilege='Get Visits' AND rp2.role=rp.role);
+            </sql>
+	</changeSet>
+
+
+    <changeSet id="TRUNK-3422-adding-Get-Encounter-Provider-Privilege" author="PralayRamteke" >
+        <preConditions  onFail="MARK_RAN">
+            <not>
+                <sqlCheck expectedResult="0">
+                SELECT count(*) FROM role_privilege  WHERE privilege='View Encounters';
+                </sqlCheck>
+            </not>
+        </preConditions>
+        <comment>
+            Add "Get Encounter Provider" privilege to the roles having "View Encounter"
+        </comment>
+            <sql>
+                INSERT INTO privilege (privilege,description)
```

Comment:
Do you really need to insert the privilege? I thought the API does it automatically unless if you saying that they're not yet inserted by the time this gets run. If it's the case, I would think this should be split into 2 changesets to be more readable where one adds the the privilege and the other associates to the appropriate roles